src/components/k8s/pcn_k8s/networkpolicies/doc.go

```diff
@@ -0,0 +1,15 @@
+// Package networkpolicies contains logic for parsing policies.
+// It parses policies from kubernetes and polycube and gets back rules in a format that the firewall can understand.
+// Then, the network policy manager, will take care of managing firewall managers inside its node
+package networkpolicies
+
+const (
+	// DPS is the short name of the Default Policy Parser
+	DPS = "Default Policy Parser"
```

Comment:
@SunSince90 Is it possible to be called as `DPM` - `Default Policy Manager`?
Just to get it aligned with other naming convention like PM and FWM

Reply:
The reason why this is called "parser" and the others are "manager" is because this does not manage anything, it just converts. Additionally, there's another object called Network Policy Manager and I don't want to create confusion, but if you feel this name would suit it better, then I'm ok with it :)

Reply:
How about DPP? Because the S in acronym of DPS stands for nothing.

Reply:
Whops, you're right :D 
Anyway, in the new version, this is called `K8s Policy Parser` (should be more descriptive), and the short name is KPP. Going to push all the final changes later today or tomorrow morning.
Thank you! :)